include/engine/api/flatbuffers/route.fbs

```diff
@@ -9,7 +9,8 @@ table Annotation {
     distance: [uint];
     duration: [uint];
     datasources: [uint];
-    nodes: [uint];
+    nodes: [ulong];
+    ways: [long];
```

Comment:
Should it be a `ulong` too?

Reply:
@akashihi I've made returned Way IDs signed to add the way direction info as explained in comments and proposed documentation. The OSM Way ID is unsigned, but Way IDs in the annotation are signed: the negative value means opposite moving direction, while positive means moving forward along the way.